### PR TITLE
UI improvement

### PR DESCRIPTION
**Pull request by Ankur9669**

### CODE REVIEW

File: FitVid/src/Components/ChipsContainer/Chip/chip.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    justify-content: center;
+    border-radius: 1rem;
+    border: 2px solid var(--primary-color);
+    width: fit-content;
+    padding: 0.5rem 2rem;
+    margin: 0.5rem;
+    cursor: pointer;
+    font-size: 1rem;
+}
+.chip-active{
```
Comment (Priya-Patel31) [inline FitVid/src/Components/ChipsContainer/Chip/chip.css]: you can integrate this class below class as they have same properties.

File: FitVid/src/Components/Video/video.css [hunk 2]
```diff
@@ -5,9 +5,27 @@
     cursor: pointer;
     background-color: var(--background-color-dark);
 }
-.videolisting-video{
+.videolisting-video-thumbnail{
+    object-fit: contain;
+}
+.videolisting-thumbnail-container{
+    position: relative;
+    width: "100%";
```
Comment (Priya-Patel31) [inline FitVid/src/Components/Video/video.css]: why you have wrapped in double quotes ?? 
It's not needed . Remove it.

File: FitVid/src/Components/Video/video.css [hunk 3]
```diff
@@ -5,9 +5,27 @@
...
+    height: "15rem";
+}
+.videolisting-thumbnail-overlay-container{
+    position: absolute;
+    top: 0px;
     width: 100%;
     height: 100%;
+    background-color: rgba(0, 0, 0, 0.8);
```
Comment (Priya-Patel31) [inline FitVid/src/Components/Video/video.css:21]: use variables to store this.

File: FitVid/src/Context/category-context.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { createContext, useContext, useState } from "react";
+// import { historyReducer } from "../Reducer/history-reducer";
```
Comment (Priya-Patel31) [inline FitVid/src/Context/category-context.js]: remove commented code .

File: FitVid/src/Pages/VideoListing/VideosContainer/VideosContainer.js [hunk 5]
```diff
@@ -2,23 +2,43 @@ import React, { useEffect, useState } from "react";
 import Video from "../../../Components/Video/Video";
 import "../videolisting.css";
 import { fetchVideos } from "../../../Util/fetch-videos";
+import { useCategory } from "../../../Context/category-context";
+import { getFilteredVideos } from "../../../Util/get-filtered-videos";
+import { useVideos } from "../../../Context/video-context";
 
 const VideosContainer = () => {
-  const [videos, setVideos] = useState([]);
+  // const [videos, setVideos] = useState([]);
```
Comment (Priya-Patel31) [inline FitVid/src/Pages/VideoListing/VideosContainer/VideosContainer.js]: remove commented code

File: FitVid/src/Pages/PageNotFound/pagenotfound.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+.page-not-found-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0px 2rem;
+}
+.not-found-image{
+    max-width: 500px;
```
Comment (Priya-Patel31) [inline FitVid/src/Pages/PageNotFound/pagenotfound.css]: change px to rem

**Review by Priya-Patel31 — APPROVED** [review]
The code seems good to me . But I have suggested few changes you can go through it .

File: FitVid/src/Components/Video/video.css (hunk 2 above)
Comment (Ankur9669) [reply FitVid/src/Components/Video/video.css]: Thanks, for pointing this out

File: FitVid/src/Context/category-context.js (hunk 4 above)
Comment (Ankur9669) [reply FitVid/src/Context/category-context.js]: Done

File: FitVid/src/Pages/VideoListing/VideosContainer/VideosContainer.js (hunk 5 above)
Comment (Ankur9669) [reply FitVid/src/Pages/VideoListing/VideosContainer/VideosContainer.js]: Done

File: FitVid/src/Pages/PageNotFound/pagenotfound.css (hunk 6 above)
Comment (Ankur9669) [reply FitVid/src/Pages/PageNotFound/pagenotfound.css]: Done

**Pull request merged**